stonesoup/sensor/base.py

```diff
@@ -19,3 +20,29 @@ class Sensor(Base):
     def gen_measurement(**kwargs):
```

Comment:
For another time: 
Can we just call it `measure(**kwargs)` or `sense(**kwargs)`? Seems cleaner, more apposite.

Reply:
modified to measure - i think it makes it clearer and it was only a few modifications. Suggest people have a look and decide if this is a change we want to make or to remain with gen_measurement